src/stability_sdk/utils.py

```diff
@@ -29,6 +29,12 @@
     
 MAX_FILENAME_SZ = int(os.getenv("MAX_FILENAME_SZ", 200))
 
+def artifact_type_to_str(artifact_type: int):
+    try:
+        return generation.ArtifactType.Name(artifact_type)
+    except ValueError:
+        return "UNKNOWN"
```

Comment:
rather than UNKNOWN, let's fall back to the default string or repl for the object.

Reply:
following up from discussion on discord, triggering situation here is the new latent tensor response type. the issue at hand is really that the SDK api-interface library is out of synch. theoretically, this shouldn't happen unless the user is using an old client, so I suggest we warn the user and recommend a client update. so maybe something like:

```python
def artifact_type_to_str(artifact_type: int):
    try:
        return generation.ArtifactType.Name(artifact_type)
    except ValueError:
        warnings.warn(
            f"Received artifact of type {artifact_type}, which is not recognized in the loaded protobuf definition.\n"
            "If you are seeing this message, you might be using an old version of the client library. Please update your client via `pip install --upgrade stability-sdk`\n"
            "If updating the client does not make this warning message go away, please report this behavior to https://github.com/Stability-AI/stability-sdk/issues/new"
)
        return f"ARTIFACT_UNRECOGNIZED"
```

regarding whether the return value is derived from the default __str__ method or is a more generic string value, I guess it's not entirely clear to me how this function might get used in the future. If the motivation is just friendlier verbose outputs, we should probably document that in the docstring